lib/mock/pending-interceptors-formatter.js

```diff
@@ -7,6 +7,9 @@ const { Console } = require('node:console')
  * Gets the output of `console.table(…)` as a string.
  */
 module.exports = class PendingInterceptorsFormatter {
+  static PERSISTENT = process.versions.icu ? '✅' : 'Y '
+  static NOT_PERSISTENT = process.versions.icu ? '❌' : 'N '
+
```

Comment:
Do we need to add these static properties?
Cant we define them outside the class?

Reply:
I imported them from test file, and given that the default export is a class, static members was cleanest. Seeing your next comment, I'll pull them up out of the class here, and re-define in test.